docs/src/CommutativeAlgebra/ca.md

```diff
@@ -306,44 +314,52 @@ A, f = quo(R, ideal(R, [x^2-y^3, x-y]))
 f
 ```
 
-### Noether Normalization
+### Data Associated to Affine Algebras
 
-```@docs
-noether_normalization(A::MPolyQuo)
-```
+#### Basic Data
 
-###### Example
+If `B = S/J` is an affine algebra, then `B.R` refers to `S` and `A.I` to `J`.
```

Comment:
Do we really want to document these internals? I'd rather expose functions to access those, then at least we stay free to change the implementation details.

Reply:
I am going to merge as is and discuss this independently. I agree that the internals of the data structures should not be there. (they should be elsewhere)

Reply:
Why is this internal. The user need to know this in order to write his own functions.